### ISSUE TITLE
Add patch validator to CMS

### ISSUE THREAD

**Issue by AlenPelin** [SitecoreSupport/Readme#6]
Thanks to #4 all patches are now signed with Sitecore certificate. Next step is to make Sitecore fail when a patch is used and not signed.

**Comment by AlenPelin** [comment]
@RomanChernyk @olegburov @ivansharamok @andrew-at-sitecore your thoughts on that?

**Comment by olegburov** [comment]
Maybe not fail, but log the `WARN` or `ERROR` message.

Do you mean to add a check into the core product, for example a processor like the `/sitecore/pipelines/initialize/processor type="Sitecore.Pipelines.Loader.ShowVersion, Sitecore.Kernel"`,
or do you mean to put something into each patch?

I like the first option if it possible, but we need to finished issue #3 first.

**Comment by AlenPelin** [comment]
Good point about warn or error message, since we have a lot of legacy patches that are not signed.

Agree that #3 must be implemented first.
